### PR TITLE
Changes for OpenJ9 0.12.1

### PR DESCRIPTION
**Pull request by SueChaplain**
To address issue https://github.com/eclipse/openj9#4530,
the Digest algorithm is no longer enabled by default,
and the system property -Djdk.nativeDigest is ignored.

Add a new topic for v0.12.1 and modify topics that
describe OpenSSL appropriately.

Signed-off-by: Sue Chaplain <sue_chaplain@uk.ibm.com>

### CODE REVIEW

**Comment by SueChaplain** [comment]
Jenkins doc stage

**Comment by SueChaplain** [comment]
Jenkins doc stage

File: docs/djdknativecrypto.md [hunk 1]
```diff
@@ -38,12 +38,12 @@ This option controls the use of OpenSSL native cryptographic support.
...
 
-OpenSSL support is enabled by default for the Digest, CBC, GCM, and RSA algorithms. If you want to turn off the OpenSSL implementation, set this option to `false`.
+OpenSSL support is enabled by default for the <!--Digest,--> CBC, GCM, and RSA algorithms. If you want to turn off the OpenSSL implementation, set this option to `false`.
 
-If you want to turn off the Digest, CBC, GCM, or RSA algorithms individually, use the following system properties:
+If you want to turn off the an algorithms individually, use the following system properties:
 
 - [`-Djdk.nativeCBC`](djdknativecbc.md)
-- [`-Djdk.nativeDigest`](djdknativedigest.md)
+<!--- [`-Djdk.nativeDigest`](djdknativedigest.md)-->
```
Comment (pshipton) [inline docs/djdknativecrypto.md]: This causes a hiccup in the bulletted list.

**Comment by SueChaplain** [comment]
Jenkins doc stage

File: docs/djdknativedigest.md [hunk 2]
```diff
@@ -26,8 +26,10 @@
 
 This option enables or disables OpenSSL native cryptographic support for the Digest algorithm.
 
+<i class="fa fa-exclamation-triangle" aria-hidden="true"></i> **Restriction:** Pending the resolution of Github issue [#4530] (https://github.com/eclipse/openj9#4530), this option is disabled and has no effect. Currently, the Digest algorithm is not enabled by default.
```
Comment (pshipton) [inline docs/djdknativedigest.md]: #4530 is already resolved, so this cannot be pending on the resolution of #4530. Also the link is incorrect, and there is a space between `] (`.

File: docs/version0.12.1.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+<!--
+* Copyright (c) 2017, 2019 IBM Corp. and others
```
Comment (pshipton) [inline docs/version0.12.1.md]: Start copyright should be 2019.

File: docs/djdknativecrypto.md (hunk 1 above)
Comment (SueChaplain) [reply docs/djdknativecrypto.md]: Yep, moved out now.

File: docs/version0.12.1.md (hunk 3 above)
Comment (SueChaplain) [reply docs/version0.12.1.md]: We view the doc as a collection, therefore the copyright is the same across all files regardless of when they were created.

**Comment by SueChaplain** [comment]
Jenkins doc stage

**Pull request merged**